releases/patch-releases.md

```diff
@@ -54,6 +54,14 @@ releases may also occur in between these.
 
 ## Detailed Release History for Active Branches
 
+### 1.21
+
+End of Life for **1.21** is **2022-04-30**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.21.1        | 2021-05-07           | 2021-05-12  |
```

Comment:
I assume that we will not cut 1.21.1 next week, since 1.21.0 went out yesterday. Is that correct or not?

cc @kubernetes/release-managers

Reply:
I agree.

Reply:
agree, we will release just 1.18/1.19/1.20